src/test/java/com/google/javascript/clutz/types_externs_with_platform.js

```diff
@@ -3,6 +3,7 @@ goog.provide('typesWithExterns.A');
 goog.provide('typesWithExterns.B');
 goog.provide('typesWithExterns.C');
 goog.provide('typesWithExterns.ArrayLike');
+goog.provide('type_renaming_with_externs');
 
 /**
```

Comment:
This is just an existing test moved to a single _with_platform.js, due to how we run our Usage tests.